Windows.Devices.WiFi/Windows.Devices.Wifi.nfproj

```diff
@@ -64,12 +64,14 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="mscorlib, Version=1.10.3.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
-      <HintPath>..\packages\nanoFramework.CoreLibrary.1.10.3-preview.7\lib\mscorlib.dll</HintPath>
+      <HintPath>packages\nanoFramework.CoreLibrary.1.10.3-preview.7\lib\mscorlib.dll</HintPath>
```

Comment:
For some reason the packages location was moved...

Reply:
Do I need to do anything or can we 'resolve'?

Reply:
Just pushed the original .nfproj file.